x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.test.tsx

```diff
@@ -283,6 +284,49 @@ describe('alerts default_config', () => {
     });
   });
 
+  describe('buildAlertsFilterByRuleIds', () => {
+    it('given an empty list of rule ids will return an empty filter', () => {
+      const filters = buildAlertsFilterByRuleIds([]);
+      expect(filters).toHaveLength(0);
+    });
+
+    it('builds filter containing all rule ids passed into function', () => {
```

Comment:
Nit: we're testing what happens when the function is given either 0 or 3 rule_ids; it would be good to add a "1" test so we have coverage for the typical "0,1,n" permutations.